0138-return-to-hnt.md

```diff
@@ -106,35 +110,33 @@ Subnetwork tokens are currently used for various supply-side operations within s
 
 ## Implementation
 
-### Phased Approach
+The implementation of this proposal requires independent changes to HNT, IOT and MOBILE, and several votes. We describe these steps below.
 
-1. Phase 1 redirects HNT emissions from HST to the MOBILE treasury. As this is a fairly straightforward change, it would require light implementation and security review. It could likely be deployed within weeks of passing the HIP.
-2. Phase 2 is the bulk of work for this proposal. It is expected to take less than three months of work, but requires significant security review and UX improvements. Users would need to claim rewards as they have done in the past, but would receive HNT instead of IOT or MOBILE. We propose an implementation date on or before 2025-01-15. It would be a goal of the core developers to ensure this phase could start as soon as possible.
-3. The remaining changes would be implemented prior to the next halvening, which is currently scheduled for August 1, 2025.
+### Voting
 
-### HST Reallocation
+Because this proposal intends to redirect HNT emissions from HST directly to participants on the network (Hotspot owners, etc.) and revoke a portion of the MOBILE premine that was passed by HIP-51 (HNT voters), it requires a veHNT vote. We propose a simple binary vote for this portion of the HIP. 
 
-With discontinued treasury emissions, MOBILE holders could lose value due to the low funding rate of the MOBILE treasury. This proposal reallocates HST emissions towards the MOBILE treasury to balance this.
+In addition, this HIP proposes to halt emissions of IOT and MOBILE tokens, which are subject to veIOT and veMOBILE governance. With the passage of the veHNT vote, HNT emissions will come to replace existing IOT and MOBILE emissions, providing rewards to IOT and MOBILE network participants, while emissions of HNT to the IOT and MOBILE treasuries will cease.
 
-By contrast, IOT holders benefit without a treasury subsidy, as their strong treasury rate is preserved when IOT emissions cease at the beginning of Phase 2.
+Therefore, veIOT and veMOBILE holders will each vote separately on the proposal to stop the emission of IOT and MOBILE tokens simultaneously with the cessation of HNT emissions to the treasuries and the begin of HNT emissions to the treasuries.
 
-With the switch to HNT in Phase 2, the value of rewards to IOT participants may decline, as they will no longer be supported by the strong IOT treasury.
+Note that veIOT or veMOBILE holders could theoretically vote to continue emission of a subnetwork token, contrary to the recommendation of this proposal, but this would result in the issuance of tokens unbacked by treasury HNT, thereby diluting the subnetwork token supply and impairing the treasury exchange rate, and thus the value in HNT terms, of the existing subnetwork tokens.
```

Comment:
```suggestion
Note that veIOT or veMOBILE holders could theoretically vote to continue emission of a subnetwork token, contrary to the recommendation of this proposal, but this would result in the issuance of tokens unbacked by treasury HNT. It's notable that this would dilute the subnetwork token supply and reduce the exchange rate, in HNT terms, of the existing subnetwork tokens.
```